### ISSUE TITLE
1

### ISSUE THREAD

**Issue by GIVEJOO** [khsoon0427/A1#2]
In [ ]: # train/test data 221
Kernel Widgets Help
си Code
STEP 회귀 모델 생성
Trusted
In [ ]: #formula (y~ X1 + X2 +...) 01.
# 모델의 생성 및 적합
A 07
df_train, dt_test = train_test_split(df_raw_dummy, test_size = 0.3, random_state
print("train data size :{}". format (df_train.shape))
print("test data size: {}". format (df_test.shape))
Logout
Python 3 (ipykernel) O
= 1234
Windows 정품 인증
(Intercept)
formula "FAT AGE+WEIGHT + HEIGHT + NECK + CHEST + ABDOMEN + HIP + THIGH + W

**Comment by GIVEJOO** [comment]
df_train_y = df_train["FAT"]
In [ ]: #statsmodels +
+
Kernel Widgets
df_train_x = df_train.drop("FAT", axis = 1)
In []: # DataFrame
In [] #df_train_x_const.values
Code
+
df_train_x_const = add_constant (df_train_x)
df_train_x_const.head()
Help
Trusted
A Q
Logout
Python 3 (ipykernel) O
df_vif = pd.DataFrame()
Windows O
df_vif["Variable"]= df_train_x_const.columns
df_vif["VIF"] = [variance_inflation_factor (df_train_x_const.values, i) W01010 Windows
for i in range(df_train_x_const.shape[1])]

**Comment by GIVEJOO** [comment]
df_train_y = df_train["FAT"]
df_train_x = df_train.drop("FAT", axis = 1)

**Comment by GIVEJOO** [comment]
df_train_x_const = add_constant (df_train_x)
df_train_x_const.head()

**Comment by GIVEJOO** [comment]
df_vif=pd.DataFrame()
df_vif["Variable"]= df_train_x_const.columns
df_vif["VIF"] = [variance_inflation_factor (df_train_x_const.values, i)
    for i in range(df_train_x_const.shape[1])]

**Comment by khsoon0427** [comment]
퍼가요
